ckanext/canada/templates/package/confirm_delete_resource.html

```diff
@@ -5,6 +5,19 @@
 {% block primary_content_inner %}
   <div class="col-md-12">
     {% block form %}
+      {% block errors %}
+        {%- set schema = h.scheming_get_dataset_schema(dataset_type) -%}
+        {% if dataset_errors %}
+          {%- snippet 'scheming/snippets/dataset_errors.html',
+            errors=dataset_errors, resource_fields=schema.resource_fields,
+            dataset_fields=schema.dataset_fields, pkg_name=pkg_id,
```

Comment:
instead of using helpers to repeatedly call package_show/resource_show in the snippet can we not pass the whole pkg_dict here?

Reply:
Yes, just have to modify the DeleteView in the core view to pass in the pkg_dict, it only passes the pkg_id and resource_dict for the current Resource, need names for the dataset and other resources. So I can do that for sure. I hated writing these helper, they felt bad haha